web/vibe/web/web.d

```diff
@@ -1020,8 +1029,10 @@ private void handleRequest(string M, alias overload, C, ERROR...)(HTTPServerRequ
 				} else {
 					res.writeBody(ret);
 				}
+			} else static if (is(RET : string)) {
+				res.writeBody(ret);
 			} else {
-				static assert(is(RET == void), M~": Only InputStream, Json and void are supported as return types for route methods.");
+				static assert(is(RET == void), M~": Only InputStream, Json, string and void are supported as return types for route methods.");
```

Comment:
Would be nice to add `const(ubyte)[]` here, too (and change to `const(char)[]` if adjusted).